src/strategies/erc20-balance-starknet/index.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'wa0x6e';
+export const version = '0.0.1';
+
```

Comment:
Missing supportedProtocols

```suggestion
export const author = 'wa0x6e';
export const version = '0.0.1';
export const supportedProtocols = ['starknet'];
```

Not sure how it is not filtering starknet addresses without this 🤔 we default it to evm addresses right?